mediapipe/pose_tracking_dll/README.md

```diff
@@ -0,0 +1,33 @@
+## Description
+The pose_tracking_dll module allows for building a Mediapipe-based pose tracking DLL library that can be used with any C++ project. All the dependencies such as tensorflow are built statically into the dll. 
+
+Currently, the following features are supported:
+- Segmenting the person(s) of interest
+- Segmenting the skeleton(s)
+- Accessing the 3D coordinates of each node of the skeleton
+
+## Prerequisites
+Follow the guidelines on the official Mediapipe website: https://google.github.io/mediapipe/getting_started/install.html#installing-on-windows
+
+IMPORTANT: The tutorial does not specify which version of Bazel to install. Install Bazel version 3.7.2
+
+## How to build
+Assuming you're in the root of the repository:
+
+```
+cd mediapipe
+bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1 pose_tracking_dll:pose_tracking_cpu
```

Comment:
missing  `--action_env PYTHON_BIN_PATH=<path to the python executable>` on Windows ( mentioned in point 9 here https://google.github.io/mediapipe/getting_started/install.html#installing-on-windows)

Reply:
Commited